### PR TITLE
668 - Remove specifics and add more general components to the architecture diagram

### PR DESCRIPTION
**Pull request by RomanovaPrime**
Remove specifics and add more general components to the architecture diagram

#### Overview

> Closes: < 668 >

< Pull Request Description >

#### Checklist

*Implementation*
- [ ] Implementation matches ticket acceptance criteria and technical notes
- [ ] Manually tested against Acceptance Criteria
- [ ] UI checked in Light / Dark mode

*Stability*
- [ ] Checked if changes affect any features and verified affected features work as expected
- [ ] Added unit tests for new code and verified existing tests work as expected

*Code quality*
- [ ] Updated `CHANGELOG.md`, `README.md` and package versions in `pubspec.yaml`
- [ ] Dependencies are updated to latest versions or new tickets are created if there are breaking changes or deprecations
- [ ] If an unrelated part of the codebase needs to be updated or refactored, create tickets with proposed changes

### CODE REVIEW

File: packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/docs/app_architecture.png
Comment (StanevPrime) [inline packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/docs/app_architecture.png:1]: image
Do we really want such high resolution ?

**Review by StanevPrime — CHANGES_REQUESTED** [review]

**Review by DDavidPrime — APPROVED** [review]

File: packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/docs/app_architecture.png
Comment (RomanovaPrime) [reply packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/docs/app_architecture.png:1]: I have reduced the image size to be similar to the previous image

**Review by StanevPrime — CHANGES_REQUESTED** [review]
Please make sure that you complete this action item

Updated CHANGELOG.md, README.md

**Comment by RomanovaPrime** [comment]
> Please make sure that you complete this action item
> 
> Updated CHANGELOG.md, README.md

Since I haven't change the file name, I believe we do not need any changes in README.md. 
As per CHANGELOG.md, I believe that this can be considered as documentation, so "Documentation and stability improvements" describes it enough. 
Do you, however, want me to add or change something?

**Comment by StanevPrime** [comment]
> > Please make sure that you complete this action item
> > Updated CHANGELOG.md, README.md
> 
> Since I haven't change the file name, I believe we do not need any changes in README.md. As per CHANGELOG.md, I believe that this can be considered as documentation, so "Documentation and stability improvements" describes it enough. Do you, however, want me to add or change something?

@DDavidPrime, @RomanovaPrime please add specifics for the documentation changes part of the change log

**Review by StanevPrime — APPROVED** [review]

**Pull request merged**